[oneMKL][spblas] Restrict features not supported by any backends (#542)
uxlfoundation/oneAPI-spec: source/elements/oneMKL/source/domains/spblas/data_types/set_coo_matrix_data.rst

   .. rubric:: Template parameters

   dataType

>       See :ref:`supported template types<onemkl_sparse_supported_types>`. Must
>       be the same type than what was used when creating the ``matrix_handle_t``.

Reviewer: ```suggestion
      See :ref:`supported template types<onemkl_sparse_supported_types>`. Must
      be the same type as was used when creating the ``matrix_handle_t``.
```

And similar elsewhere.

Author: Thank you, I missed many references that I meant to change as well!
This is fixed in https://github.com/uxlfoundation/oneAPI-spec/pull/542/commits/1fdd42d6cb0c400b1ff7c4821a63d6bff1c79d4b
